.env.example

```
@@ -29,0 +1,86 @@
+# =============================================================================
+# APPLICATION CONFIGURATION
+# =============================================================================
+NODE_ENV="development"
+PORT="3001"
+FRONTEND_URL="http://localhost:5000"
+
+# =============================================================================
+# SESSION CONFIGURATION - Generate secure secret
+# =============================================================================
+# Generate with: node -e "console.log(require('crypto').randomBytes(32).toString('hex'))"
+SESSION_SECRET="your_session_secret_here_32_chars_minimum"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)